kiwi/builder/live.py

```diff
@@ -240,10 +242,11 @@ def create(self):
             if self.firmware.efi_mode() == 'uefi':
                 # write bootloader config to EFI directory in order to allow
                 # grub loaded by shim to find the config file
-                shutil.copy(
-                    self.media_dir + '/boot/grub2/grub.cfg',
-                    self.media_dir + '/EFI/BOOT'
+                grubconf_glob = os.sep.join(
+                    [self.media_dir, 'boot/grub*/grub.cfg']
```

Comment:
This can not happen here, because the glob search runs on the self created temporary media_dir. The decision for grub2 or grub has been made in the prior bootloader class and there we already respect the order as you suggested in a former conversation